src/teleop/gui/src/static/rover_msgs.json

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
Is this file still relevant? I'm not exactly sure what this is used for. Is it for the gui lcm_echo and lcm_send pages?

Reply:
This file is another leftover from the old lcm_send page, but it does bring up something I wanted to discuss at our next leadership meeting. I think a full rewrite of the topic send/echo pages is in order so that a file like this is not neccesary. The way that both of these pages were originally written uses a ton of repeated code and I think a better approach now that we are using ros is to make it more of a wrapper for the rostopic command line tool. Should definitely discuss tomorrow.

Reply:
sounds good. also i'm not sure if there's a software leadership meeting tomorrow